deploy/charts/harvester/templates/configmap.yaml

```diff
@@ -145,3 +145,94 @@ data:
       sleep 2
     done
     `}}
+  cpu-manager.sh: |-
+    function cleanup_cpu_manager_state() {
+      if [ -f "$CPU_MANAGER_STATE_FILE" ]; then
+        mv "$CPU_MANAGER_STATE_FILE" "${CPU_MANAGER_STATE_FILE}.old"
+        echo "File $CPU_MANAGER_STATE_FILE has been renamed to ${CPU_MANAGER_STATE_FILE}.old"
+      else
+        echo "File $CPU_MANAGER_STATE_FILE does not exist."
+      fi
+    }
+
+    function manage_service() {
+      local service=$1
+
+      echo "Stopping ${service}."
+      if ! chroot $HOST_DIR systemctl stop $service; then
+        echo "Error: failed to stop ${service}."
+        exit 1
```

Comment:
if each none-zero return can have a different return code (1,2,3...), it is better for troubleshooting

Reply:
gentle ping @w13915984028, all comments are addressed in https://github.com/harvester/harvester/pull/6517, please take a look, thanks.